### ISSUE TITLE
Isn't working with litra-oversight v2.2.2

### ISSUE THREAD

**Issue by csytsma** [timrogers/litra#287]
After selecting the 'litra-oversight.js' for Execute and enabling Pass Arguments, it never turns the Logitech Litra light on.

Troubleshooting Steps:
- Confirmed executing litra-oversight from the command line was able to turn on/off the light
- Selected test bash script in home directory, and confirmed it got executed by OverSight when camera activated
- Created local link in home directory to litra-oversight.js file, but didn't make a difference.

I installed globally, per the instructions with Raycast. It is installed in:
/Users/<username>/.nvm/versions/node/v16.20.2/lib/node_modules/litra/dist/commonjs/cli/litra-oversight.js

Not sure if this is OverSight or Litra. Great javascript library by the way!

MacOS 13.4
OverSight v2.2.2
litra (installed today Dec 12, 2023)

**Comment by timrogers** [comment]
@csytsma Hey! 👋 Let me try this with Oversight myself and get back to you.

**Comment by csytsma** [comment]
It's really strange, can't figure it out. I created a bash script in my home directory, and in it I launch Safari and call litra-oversight.  If I execute the script from the terminal, Safari launches and the light turns on. But if Oversight calls the same script, it launches Safari, but doesn't doesn't turn on the light. My guess would be permissions?

**Comment by timrogers** [comment]
That sounds likely to me, although I'm not sure on the best way to fix it. I wonder if Oversight would log an error anywhere?

**Comment by csytsma** [comment]
I was able to finally get it working, but still with a slight workaround.  I added log output commands for the script, and saw that it was failing on finding 'node'. Strange, as from the terminal, it has no problem.

So I hardcoded all the paths to 'node' and 'litra-oversight.js' in my bash script that I call from Oversight, and now it works:
`/Users/<username>/.nvm/versions/node/v16.20.2/bin/node /Users/<username>/node_modules/litra/dist/commonjs/cli/litra-oversight.js -device $D -event $E`

I don't like hardcoding it, but it seems Oversight couldn't find the files otherwise.
